src/app/organization/organization-routing.module.ts

```diff
@@ -171,6 +173,14 @@ const routes: Routes = [
               path: 'create',
               component: CreatePaymentTypeComponent,
               data: { title: extract('Create Payment Type'), breadcrumb: 'Create Payment Type'}
+            },
+            {
+              path: 'edit/:id',
```

Comment:
It would probably be better if id comes before edit, to ensure consistency with all the other components. Also, it would be better to implement ':id' as a route on its own, and then have 'edit' as its child, as we will need this later to implement the viewPaymentTypes component. Push the data, resolve and component into the newly created 'edit' child route.

Reply:
See system>account-number-preference as reference.

Reply:
Regarding your query on gitter just now, see system>account-number-preference to see how you should deal with breadcrumbs.

Reply:
I apologise for that. You are correct, we don't need viewPaymentType in this component. Wait for Abhay's comment on this but I'd add the ID into the bread crumbs so that it becomes `Edit #ID` as we can't have a separate ID in the breadcrumb which redirects to nothing.